### PR TITLE
Enablement TCE Table Pre-Allocation for Dynamic Drawer Add

### PR DESCRIPTION
**Pull request by SteffiAntony**
Enabled TCE Table Pre-Allocation for Dynamic Drawer Add 
GUI JIRA: https://jsw.ibm.com/browse/PFEBMC-1500

### CODE REVIEW

File: src/store/modules/ResourceManagement/ResourceMemoryStore.js [hunk 1]
```diff
@@ -211,6 +248,33 @@ const ResourceMemoryStore = {
...
+        .then(() => {
+          commit(
+            'setDynamicIoDrawerCapacity',
+            updatedIoDynamicCapacity.Attributes
+              .hb_storage_preallocation_for_drawer_attach
+          );
+          return i18n.t('pageMemory.toast.successSavingAdapterDynamicCapacity');
+        })
+        .catch((error) => {
+          console.log('error', error);
```
Comment (Renuka9527) [inline src/store/modules/ResourceManagement/ResourceMemoryStore.js]: Just Add console.log(error) here

**Review by Renuka9527 — CHANGES_REQUESTED** [review]
Attach the screenshots and change the header to  Enablement TCE Table Pre-Allocation for Dynamic Drawer Add

**Comment by SteffiAntony** [comment]
> Attach the screenshots and change the header to Enablement TCE Table Pre-Allocation for Dynamic Drawer Add

Screen shot for Everest and Rainer
Screenshot 2024-02-06 at 11 22 39 AM

Screen shot for Bonnell
Screenshot 2024-02-06 at 11 29 31 AM

**Review by Renuka9527 — APPROVED** [review]

**Review by vedangimittal — APPROVED** [review]

**Review by nabilpandiyalayil — APPROVED** [review]

File: src/locales/en-US.json [hunk 2]
```diff
@@ -1061,6 +1061,9 @@
     "updateLogicalMemoryPageSetup": "Update system memory page setup",
     "updateLogicalMemorySize": "Update logical memory block size",
     "updatePageSetup": "Update system memory page setup",
+    "dynamicIoDrawerAttachmentTitle":"Dynamic I/O Drawer Attachment",
```
Comment (Nikhil-Ashoka) [inline src/locales/en-US.json]: Rename as "Dynamic I/O drawer attachment" to be consistent with the rest of the content in the page.

File: src/locales/en-US.json [hunk 3]
```diff
@@ -1061,6 +1061,9 @@
...
+    "dynamicIoDrawerAttachment": "This option allocates platform memory during IPL for PCI-E slots to enable dynamic I/O drawer attachment.",
+    "updateDynamicIoDrawerAttachment": "Update Dynamic I/O Drawer Attachment",
```
Comment (Nikhil-Ashoka) [inline src/locales/en-US.json]: Rename as "Update dynamic I/O drawer attachment".

File: src/locales/en-US.json [hunk 4]
```diff
@@ -1079,7 +1082,9 @@
       "successSavingAdapterEnlargedCapacity": "Successfully updated I/O adapter enlarged capacity",
       "successSavingLogicalMemory": "Successfully updated logical memory block size",
       "successSavingActiveMemoryMirroringMode": "Successfully updated active memory mirroring mode",
-      "successSavingPageSetup": "Successfully updated system memory page setup"
+      "successSavingPageSetup": "Successfully updated system memory page setup",
+      "successSavingAdapterDynamicCapacity": "Successfully updated I/O adapter Drawer Attachment",
```
Comment (Nikhil-Ashoka) [inline src/locales/en-US.json]: Rename as "Successfully updated I/O adapter drawer attachment"

File: src/locales/en-US.json [hunk 5]
```diff
@@ -1079,7 +1082,9 @@
...
+      "errorSavingAdapterDynamicCapacity": "Error updating I/O adapter Drawer Attachment"
```
Comment (Nikhil-Ashoka) [inline src/locales/en-US.json]: Rename as "Error updating I/O adapter drawer attachment"

**Review by Nikhil-Ashoka — APPROVED** [review]

**Pull request merged**